src/components/Register/RegisterForm/styles.module.css

```diff
@@ -0,0 +1,46 @@
+.main {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  min-width: 420px;
+}
+
+@media (max-width: 619px) {
+  .main {
+    min-width: 400px;
+  }
+}
+
+@media (max-width: 400px) {
+  .main {
+    min-width: 269px;
+  }
+}
+
+/* Overwrite form-group styling */
+.formGroupCustom {
+  margin-top: 8px;
+  margin-bottom: 0px;
+}
+
+.formGroupCustom label {
+  padding-left: 13.5px;
+}
+
+.floatLabel {
+  opacity: 1;
+  position: relative;
+  top: 40px;
+  transform: translateY(-40px);
+  transition-duration: 0.3s;
+}
+
+.noLabel {
+  opacity: 0;
+  position: relative;
+  top: 40px;
+  transform: translateY(0px);
+  transition-duration: 0.3s;
+}
```

Comment:
A very nice touch to have the labels jump above as the user starts to type. 👍

Reply:
Thank you, i felt pretty proud about that one